### ISSUE TITLE
Just a thanks!

### ISSUE THREAD

**Issue by tthsympa** [megahertz/react-simple-wysiwyg#32]
Hey man, quick message to thank you for this lib! I was looking for a simple, light and not-uber-over-engineered richText editor and it was quite hard to find what I want.

Yours is just what I needed and I even could style it to fit my design.

Keep up the good work!

**Comment by megahertz** [comment]
Thank you for your kind words @tthsympa. I'm glad that this lib is helpful for your project.

**Comment by HarperTime** [comment]
Same!

**Comment by alexberezin** [comment]
I join, thank you!

**Comment by accountunavaible** [comment]
same here! Big Thanks!

**Comment by adam-risk** [comment]
Yup, same here. Great component. Love what you've created.

**Comment by fweffort** [comment]
Same here! thanks!!

**Comment by sheam** [comment]
Agreed, I was looking for something like this after react-rte became abandoned and blocking us from moving to react 19. Which brings me to the question, will react 19 eventually be supported?

**Comment by megahertz** [comment]
@sheam It works fine with React 19

**Comment by IsaacInsoll** [comment]
Hi, 
I experimented with a few as we just have very basic requirements (paragraphs, bold, links etc) and everything else was intense overkill, this is great. 

Might as well put this here in case it helps anyone else:

**Convert to "proper" HTML with paragraphs**
```javascript
const opts = {
  ALLOWED_TAGS: ['b','em','strong','div','p','br','i','u','li','ol','ul','a'],
  ALLOWED_ATTR: [],
};

export const exportHtml = (html: string) => {
  //https://github.com/megahertz/react-simple-wysiwyg/issues/47 says process the HTML to replace DIV with P tags
  let str = DOMPurify.sanitize(html, opts)
    .replaceAll('<div', '<p')
    .replaceAll('</div', '</p');
  if (!str.startsWith('<p>')) {
    str = '<p>' + str.replace('<p>', '</p><p>');
  }
  return DOMPurify.sanitize(str, opts);
};
```
